### ISSUE TITLE
Proposal: Combine autovizwidget, sparkmagic and hdijupyterutils into one conda-recipe

### ISSUE THREAD

**Issue by ericdill** [conda-forge/sparkmagic-feedstock#8]
Problem: Building conda-forge packages for sparkmagic is a pain because the three libraries (sparkmagic, hdijupyterutils and autovizwidget) are in three different feedstocks and require three sequential PRs.

Proposal: Combine these three packages into the one meta.yaml and make releasing easier. Archive the hdijupyterutils-feedstock and the autovizwidget-feedstock. There will be no change to end users as the three conda packages will still be available independently on anaconda.org/conda-forge

Thoughts @itamarst ?

**Comment by ericdill** [comment]
oh cc @danielfrg too

**Comment by ericdill** [comment]
My plan is to get one more 0.12.8 release out for hdijupyterutils and sparkmagic (autovizwidget is already at 0.12.8) and then work on unifying these into one recipe, unless there are objections from @danielfrg or @itamarst , though if there are, I think we shuold be able to agree on a better path forward than three feedstocks :)

**Comment by danielfrg** [comment]
> oh cc @danielfrg too

Don't forget about me ERIC!!!

I think that's a better way to do it and don't see any problems. I also hated from the beginning that weird dependency cycle/whatever it is, I just did it to match convention i guess. Just out of curiosity is there any feedstocks that combine multiple recipes.

**Comment by ericdill** [comment]
> I just did it to match convention i guess

I don't think that conda-build had the multiple outputs capability when you made these feedstocks.

> Just out of curiosity is there any feedstocks that combine multiple recipes.

Lots. [Matplotlib](https://github.com/conda-forge/matplotlib-feedstock/blob/master/recipe/meta.yaml#L69-L82), [airflow](https://github.com/conda-forge/airflow-feedstock/blob/master/recipe/meta.yaml#L22-L455) to name a few. The airflow recipe is kind of hilarious, tbh. It's got like 30 outputs. I believe their intent was to match the `pip install airflow[feature]` syntax by having it be `conda install airflow-feature -c conda-forge`, though they maybe over-rotated a bit :grin:

**Comment by itamarst** [comment]
Sounds fine to me.

**Comment by itamarst** [comment]
Who has authority to get rid of the other two feedstocks? Just need to make sure they're deleted immediately after the PR gets merged, to prevent weirdness (I'm hoping to do a 0.12.9 release today).

**Comment by ericdill** [comment]
I'm on conda-forge/core so I can archive the feedstock
